src/js/binaryen.js-post.js

```diff
@@ -911,34 +911,12 @@
     return Module['_BinaryenSetAPITracing'](on);
   };
 
-  return Module;
-};
+  // Always expose globally because check.py generates a.js which requires 'Binaryen' as a global var
```

Comment:
if this is just for check.py, perhaps we could do some hack in there, just for testing? seems a shame to instrument the official production build for testing.